data/human/free worlds 0 prologue.txt

```diff
@@ -460,94 +464,168 @@ mission "FW Recon 3B"
 				"Gunboat"
 	
 	on visit
+		set "fw recon: obvious gunboat"
 		dialog `A minute after you land, a Republic Gunboat, the <npc>, lands directly next to you. JJ does not approach you. You notice from your ship's logs that this Gunboat has been following you ever since you left <origin>; before you can hand over the information, you will need to escape its pursuit.`
-	
 	on complete
-		"assisted free worlds" ++
-		dialog `JJ meets you as soon as you land, and you pass off your information and photos to him. He pays you <payment> and says, "I'll be sure to let the Free Worlds Council know how helpful you have been."`
 		payment 80000
+		"assisted free worlds" ++
 		log "Gave the Free Worlds some photos and information on the Navy base on New Wales. Jean-Jacques, the militia contact, was very grateful for the help."
+		conversation
+			`JJ meets you as soon as you land. "Good job evading that Navy Gunboat," he says.`
+			branch caught
+				has "fw recon: caught"
+			branch obvious
+				has "fw recon: obvious gunboat"
+			
+			choice
+				`	"Gunboat? What Gunboat?"`
+				`	"I knew I had a feeling that I was being followed."`
+					goto followed
+			`	"You didn't notice? I have scouts that report that you were being followed by a Gunboat by the name of the <npc> ever since you left <origin>. Keeping an eye on where you were going, no doubt.`
+				goto followed
+			
+			label caught
+			choice
+				`	"They knew I was gathering information on the base. It's no wonder they had someone following me."`
+					goto followed
+				`	"How did they know I was involved with you?"`
+					goto followed
+			
+			label obvious
+			choice
+				`	"I knew I had a feeling that I was being followed."`
+				`	"How did they know I was involved with you?"`
+			
+			label followed
+			`	"The Republic's intelligence gathering capabilities are vast, far beyond anything we can muster. I can only guess how they knew you were involved and kept an eye on you."`
+			choice
+				`	"What should I do now that the Republic is watching me?"`
+				`	"Knowing that, how can you possibly expect to stand up against the Republic?"`
+					goto odds
+			`	"Keep your head low. You're compromised, so I can't ask you to do any more recon work for us. So long as you don't cause them any trouble, they'll likely eventually leave you alone. We have a great number of merchants helping us, and as capable as the Republic is, they can only keep a close eye on so many people at a time."`
```

Comment:
The Republic is a singular entity, so saying "the Republic is" makes sense to me. But I don't see it as incorrect to mix talking about the Republic in singular while at the same time talking about the agents of the Republic in plural. Since "the Republic" is singular, saying "the Republic are" would be wrong, but if we were to change "they" to "it" then it becomes a very impersonal discussion. "It" is a pronoun you use for something you can't relate to or something that isn't living. "They" makes it more clear that there are people behind the Republic that are of concern, as opposed to treating the Republic as a monolith. So really, I think the use of "they" vs "it" comes down more to the character speaking, and I'd say that JJ is a character who makes more sense to use "they" when referring to "the Republic," although you might see "it" coming from a member of the Wolf Pack.

Reply:
If you are using "the republic" as shorthand for some defined subset of people who perform a function of the republic, then the use of "they" is appropriate. i.e. 
"Gosh, I swear every time I turn around the Republic (intelligence services) has tapped another one of our communicators and we gotta cut it off. Wish they'd manage to miss something once in a while."
vs
"As capable as the Republic (combination of repub and navy intelligence, the navy itself, local security forces, hyperspace relay regulators/monitors, local security forces and also paid mercenaries - all grouped up as the combined apparatus of the republic) **is**, **it** can only keep a close eye on so many people at a time."`